proto/remote_asset.proto

```diff
@@ -11,6 +11,10 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
+// Prevent buildfix.sh from clang-formatting this file because it's misformatted
+// in the upstream repository.
+// NO-CLANG-FORMAT
```

Comment:
nit: it looks like clang supports comments like `// clang-format off`, at least on my build of clang (10.0.0)

```suggestion

// clang-format off
```

Reply:
Ah, didn't notice that, thanks!